libnavui-dropin/src/main/res/layout-land/mapbox_navigation_view_layout.xml

```diff
@@ -107,7 +123,7 @@
                     android:layout_height="wrap_content"
                     android:gravity="top|right"
                     android:layout_marginEnd="8dp"
-                    app:layout_constraintTop_toTopOf="@id/speedLimitLayout"
+                    app:layout_constraintTop_toBottomOf="@id/compassLayout"
```

Comment:
This is not always the case. We will have to change the constraints at runtime. 
`ActionListLayout` will be 
##### When in FreeDrive (maneuver view is not visible)
`constraintTop_toBottomOf="@id/compassLayout"` - when compass is visible. We have issues with marginTop which we can look at separately.
`constraintTop_toTopOf="parent"` with `marginTop=8dp` - when compass is gone
##### When maneuver view is visible
`constraintTop_toBottomOf="@id/compassLayout"` - when compass is visible. We have issues with marginTop which we can look at separately.
`constraintTop_toTopOf="@id/guidanceLayout"` `without marginTop` - when compass is gone

This is only applicable to landscape mode.

Reply:
Same as above: it works as expected because `compassLayout` becomes GONE.
`constraintTop_toTopOf="@id/guidanceLayout"` is the same as `constraintTop_toTopOf="parent"`.
Not sure about the margins, let's discuss [here](https://github.com/mapbox/mapbox-navigation-android/pull/6355#discussion_r977324375).